core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -807,6 +807,12 @@ public MetricKey build() {
           .setMetricType(MetricType.COUNTER)
           .setIsClusterAggregated(false)
           .build();
+  public static final MetricKey WORKER_BLOCKS_EVICTED_RATE =
```

Comment:
nit: EVICTION_RATE

Reply:
I was torn between naming this `WORKER_BLOCK_EVICTION_RATE` and `WORKER_BLOCKS_EVICTED_RATE`, I can make it `WORKER_BLOCKS_EVICTION_RATE` if we don't care about the phonetic reading too?

Reply:
Both works but I guess I can't see why making it eviction_rate is a careless act phonetically :/

Reply:
Haha actually it'd depend on how you interpret the whole phrase so actually it could work too.

I thought of another possible reason to keep it as is, keeping `WORKER_BLOCKS_EVICTED` as a prefix lets you auto-complete both the original `Counter` and also the new `Meter` metrics. I think it'd make sense to keep the two as close as possible in naming since they are literally updated at the same location.